package.json

```diff
@@ -151,6 +151,15 @@
   "overrides": {
     "publish-please": {
       "lodash": "4.17.21"
+    },
+    "@lando/compose": {
+      "js-yaml": "^4.1.0"
+    },
+    "@lando/mailhog": {
+      "js-yaml": "^4.1.0"
+    },
+    "@lando/phpmyadmin": {
+      "js-yaml": "^4.1.0"
```

Comment:
Lando plugins do not use `js-yaml`, it is a dead dependency. We cannot remove that without forking, but at least we can set it to the version we use to avoid multiple versions of `js-yaml`.